src/backend/distributed/transaction/backend_data.c

```diff
@@ -780,6 +784,85 @@ MarkCitusInitiatedCoordinatorBackend(void)
 }
 
 
+/*
+ * AssignGlobalPID assigns a global process id for the current backend
+ */
+void
+AssignGlobalPID(void)
+{
+	if (MyBackendData->globalPID != 0)
+	{
+		elog(DEBUG1, "a global pid is already assigned to this backend");
```

Comment:
maybe print the globalId as well?And probably DEBUG4 is more convenient for such a low level information

Reply:
As I explained in the previous comment, I think we should remove this message since we decided to call it every time we call get gpid function